gestalt-asset-core/src/test/java/org/terasology/assets/module/autoreload/ModuleEnvironmentWatcherTest.java

```diff
@@ -108,7 +108,17 @@ public void test() throws IOException {
         }
         changed = watcher.checkForChanges();
         assertTrue(changed.containsEntry(assetType, new ResourceUrn(module.getId(), new Name("test.txt"))));
-        FilesUtil.recursiveDelete(tempDirectory);
+        try {
+            FilesUtil.recursiveDelete(tempDirectory);
+        } catch (IOException e) {
+            // Observed on Windows 10 to fail - timing issue? Try again after a brief moment!
```

Comment:
I haven't had this fail under Windows 10. It is a bit odd.... perhaps you have a virus scanner or something that is locking and scanning the directory after creation?

Probably would suggest we rework it into a small look with pause and max attempts, and build that into the method.

Reply:
Quite possible. I also still haven't gotten the branch working at all on my Mac. Will keep at it and not merge till I can validate it on 2-3 different systems (I've got a second Win10 system I can test too)